Fix grammar test (#272)
flyteorg/flytectl: docs/source/gen/flytectl_create_project.rst

flytectl create project
-----------------------

Creates project resources

Synopsis
~~~~~~~~

Creates projects:

::

 flytectl create project --name flytesnacks --id flytesnacks --description "flytesnacks description"  --labels app=flyte

.. note::

>    The term project/projects can be used interchangeably in these commands.

Reviewer: ```suggestion
   The terms project/projects are interchangeable in these commands.
```
